src/utils/backend/Backoff.js

```diff
@@ -17,8 +18,13 @@ class Backoff {
       ms *= rand
     }
     this.ms = this.ms * this.factor
+    if (this.ms >= this.max) this.ms = this.opts.min || 100
     return Math.min(ms, this.max)
   }
+
+  reset() {
+    this.ms = this.opts.min || 100
```

Comment:
Is think `this.opts.min || 100` is not necessary since we already set it in the constructor.